host_core/lib/application.ex

```diff
@@ -39,76 +41,40 @@ defmodule HostCore do
     started
   end
 
+  def host_count() do
+    Registry.count(Registry.HostRegistry)
+  end
+
+  defp create_ets_tables() do
+    :ets.new(:vhost_table, [:named_table, :set, :public])
+    :ets.new(:policy_table, [:named_table, :set, :public])
```

Comment:
policy is currently using extra data as part of the key. It's possible to refactor it to use an isolated table, but since the policy cache decisions are isolated by opaque guid, it's okay for now.